### PR TITLE
🎨 Improve LulzBot FTDI Eve Touch UI

### PR DESCRIPTION
**Pull request by bmkahl**
### Description
This PR updates the FTDI Eve Touch UI that the LulzBot TAZPro uses. 

### Requirements
Use TOUCH_UI_FTDI_EVE

### Benefits

These changes make it more user friendly to use and update the look of the menus.

### CODE REVIEW

**Comment by thinkyhead** [comment]
There are parts missing so it can't build. Can you provide the missing pieces?

File: Marlin/src/lcd/extui/ftdi_eve_touch_ui/ftdi_eve_lib/extended/event_loop.cpp [hunk 1]
```diff
@@ -117,6 +119,28 @@ namespace FTDI {
...
+    switch (ExtUI::pauseModeStatus) {
+      case PAUSE_MESSAGE_PARKING:  ExtUI::onStatusChanged(GET_TEXT_F(MSG_PAUSE_PRINT_PARKING)); break;
+      case PAUSE_MESSAGE_CHANGING: ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_INIT)); break;
+      case PAUSE_MESSAGE_UNLOAD:   ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_UNLOAD)); break;
+      case PAUSE_MESSAGE_WAITING:  ExtUI::onStatusChanged(GET_TEXT_F(MSG_ADVANCED_PAUSE_WAITING)); break;
+      case PAUSE_MESSAGE_INSERT:   ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_INSERT)); break;
+      case PAUSE_MESSAGE_LOAD:     ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_LOAD)); break;
+      case PAUSE_MESSAGE_PURGE:    ExtUI::onStatusChanged(GET_TEXT_F(TERN(ADVANCED_PAUSE_CONTINUOUS_PURGE, MSG_FILAMENT_CHANGE_CONT_PURGE, MSG_FILAMENT_CHANGE_PURGE))); break;
+      case PAUSE_MESSAGE_RESUME:   ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_RESUME)); break;
+      case PAUSE_MESSAGE_HEAT:     ExtUI::onStatusChanged(GET_TEXT_F(MSG_FILAMENT_CHANGE_RESUME_REHEAT)); break;
```
Comment (thinkyhead) [inline Marlin/src/lcd/extui/ftdi_eve_touch_ui/ftdi_eve_lib/extended/event_loop.cpp]: We now use "`MSG_FILAMENT_CHANGE_HEAT`" here. There may be some other updates that Lulzbot hasn't brought over.

**Comment by thinkyhead** [comment]
We would like to make edits and corrections, but this PR was not submitted from the same account as the source fork, so we are unable to make the changes. If you make a fork of this under @bmkahl then we will be able to edit a new PR submitted from there. The other alternative would be for me to move this branch to my own fork, but then you would be required to submit PRs to that branch if you had anything to add. So, it is better if you put this in your own fork for full control and collaboration.

**Comment by bmkahl** [comment]
Will do

**Pull request closed without merging**